### ISSUE TITLE
Errors running script `write_table_for_TMX_alt.groovy` on the command line

### ISSUE THREAD

**Issue by msoutopico** [kosivantsov/omegat_scripts#14]
The script runs well on the GUI (tested with version 4.3.2) but on the command line (on a linux server) there are some errors.

This is the line I use:

```
java -jar /opt/omegat/OmegaT_4.3.2_Linux_64/OmegaT.jar projectDir --script=/opt/omegat/scripts/write_table_for_TMX_alt.groovy --mode=console-translate
```

This is my java:
```
openjdk version "11.0.13" 2021-10-19
OpenJDK Runtime Environment (build 11.0.13+8-Ubuntu-0ubuntu1.20.04)
OpenJDK 64-Bit Server VM (build 11.0.13+8-Ubuntu-0ubuntu1.20.04, mixed mode, sharing)
```

And these are the errors that I get:
```
19551: Info: Project loading end (LOG_DATAENGINE_LOAD_END)
19551: Info: Executing script /opt/omegat/scripts/write_table_for_TMX_alt.groovy (event LOAD)
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.vmplugin.v7.Java7$1 (file:/opt/omegat/OmegaT_4.3.2_Linux_64/lib/groovy-2.5.6.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int)
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.vmplugin.v7.Java7$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
19551: Error: javax.script.ScriptException: javax.script.ScriptException: java.lang.NullPointerException: Cannot invoke method getCurrentFile() on null object
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:158)
19551: Error:   at java.scripting/javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)
19551: Error:   at org.omegat.gui.scripting.ScriptRunner.executeScript(ScriptRunner.java:169)
19551: Error:   at org.omegat.gui.scripting.ScriptRunner.executeScript(ScriptRunner.java:133)
19551: Error:   at org.omegat.gui.scripting.ScriptRunner.executeScript(ScriptRunner.java:85)
19551: Error:   at org.omegat.Main.executeConsoleScript(Main.java:552)
19551: Error:   at org.omegat.Main.selectProjectConsoleMode(Main.java:526)
19551: Error:   at org.omegat.Main.runConsoleTranslate(Main.java:335)
19551: Error:   at org.omegat.Main.main(Main.java:187)
19551: Error: Caused by: javax.script.ScriptException: java.lang.NullPointerException: Cannot invoke method getCurrentFile() on null object
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:320)
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:155)
19551: Error:   ... 8 more
19551: Error: Caused by: java.lang.NullPointerException: Cannot invoke method getCurrentFile() on null object
19551: Error:   at org.codehaus.groovy.runtime.NullObject.invokeMethod(NullObject.java:91)
19551: Error:   at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:43)
19551: Error:   at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
19551: Error:   at org.codehaus.groovy.runtime.callsite.NullCallSite.call(NullCallSite.java:34)
19551: Error:   at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
19551: Error:   at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:115)
19551: Error:   at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:119)
19551: Error:   at Script1.run(Script1.groovy:94)
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:317)
19551: Error:   ... 9 more
Translating Project
19551: Info: Project compiling start (LOG_DATAENGINE_COMPILE_START)
Creating translated documents: ft2ms_alt-trans_test.docx
Translated document created
19551: Info: Project compiling end (LOG_DATAENGINE_COMPILE_END)
19551: Info: Executing script /opt/omegat/scripts/write_table_for_TMX_alt.groovy (event COMPILE)
19551: Info: Event: project change - "COMPILE" (LOG_INFO_EVENT_PROJECT_CHANGE)
19551: Error: javax.script.ScriptException: javax.script.ScriptException: java.lang.NullPointerException: Cannot invoke method getCurrentFile() on null object
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:158)
19551: Error:   at java.scripting/javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)
19551: Error:   at org.omegat.gui.scripting.ScriptRunner.executeScript(ScriptRunner.java:169)
19551: Error:   at org.omegat.gui.scripting.ScriptRunner.executeScript(ScriptRunner.java:133)
19551: Error:   at org.omegat.gui.scripting.ScriptRunner.executeScript(ScriptRunner.java:85)
19551: Error:   at org.omegat.Main.executeConsoleScript(Main.java:552)
19551: Error:   at org.omegat.Main.runConsoleTranslate(Main.java:350)
19551: Error:   at org.omegat.Main.main(Main.java:187)
19551: Error: Caused by: javax.script.ScriptException: java.lang.NullPointerException: Cannot invoke method getCurrentFile() on null object
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:320)
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:155)
19551: Error:   ... 7 more
19551: Error: Caused by: java.lang.NullPointerException: Cannot invoke method getCurrentFile() on null object
19551: Error:   at org.codehaus.groovy.runtime.NullObject.invokeMethod(NullObject.java:91)
19551: Error:   at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:43)
19551: Error:   at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
19551: Error:   at org.codehaus.groovy.runtime.callsite.NullCallSite.call(NullCallSite.java:34)
19551: Error:   at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
19551: Error:   at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:115)
19551: Error:   at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:119)
19551: Error:   at Script2.run(Script2.groovy:94)
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:317)
19551: Error:   ... 8 more
19551: Info: Project closed (LOG_DATAENGINE_CLOSE)
19551: Info: Executing script /opt/omegat/scripts/write_table_for_TMX_alt.groovy (event CLOSE)
19551: Error: javax.script.ScriptException: javax.script.ScriptException: java.lang.NullPointerException: Cannot invoke method getCurrentFile() on null object
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:158)
19551: Error:   at java.scripting/javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)
19551: Error:   at org.omegat.gui.scripting.ScriptRunner.executeScript(ScriptRunner.java:169)
19551: Error:   at org.omegat.gui.scripting.ScriptRunner.executeScript(ScriptRunner.java:133)
19551: Error:   at org.omegat.gui.scripting.ScriptRunner.executeScript(ScriptRunner.java:85)
19551: Error:   at org.omegat.Main.executeConsoleScript(Main.java:552)
19551: Error:   at org.omegat.Main.runConsoleTranslate(Main.java:353)
19551: Error:   at org.omegat.Main.main(Main.java:187)
19551: Error: Caused by: javax.script.ScriptException: java.lang.NullPointerException: Cannot invoke method getCurrentFile() on null object
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:320)
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:155)
19551: Error:   ... 7 more
19551: Error: Caused by: java.lang.NullPointerException: Cannot invoke method getCurrentFile() on null object
19551: Error:   at org.codehaus.groovy.runtime.NullObject.invokeMethod(NullObject.java:91)
19551: Error:   at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:43)
19551: Error:   at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
19551: Error:   at org.codehaus.groovy.runtime.callsite.NullCallSite.call(NullCallSite.java:34)
19551: Error:   at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
19551: Error:   at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:115)
19551: Error:   at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:119)
19551: Error:   at Script3.run(Script3.groovy:94)
19551: Error:   at org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.eval(GroovyScriptEngineImpl.java:317)
19551: Error:   ... 8 more
Finished
```

Let me know if any more info can be useful.

Thank you, Kos.

**Comment by kosivantsov** [comment]
Should be fixed as of 6d29204ba2121f05c2608c075441dc6410b71bd1

**Comment by kosivantsov** [comment]
Could you confirm?
